### PR TITLE
Limit permissions

### PR DESCRIPTION
**Pull request by izar**
Trying out security advice by OSSF Scorecard

### CODE REVIEW

**Comment by colesmj** [comment]
@izar I'm okay approving, but please confirm the automated checks outcomes are expected (2 failed, 2 cancelled).

**Comment by izar** [comment]
Yeah I'm not sure what happened there - trying to re-trigger those tests.

**Comment by colesmj** [comment]
Approved, pending automated checks

**Review by colesmj — APPROVED** [review]

File: SECURITY.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+Reporting a Vulnerability
+Please report (suspected) security vulnerabilities as a project issue. You will receive a response from us within 48 hours. If the issue is confirmed, we will release a patch as soon as possible depending on complexity but historically within a few days.
```
Comment (colesmj) [inline SECURITY.md:2]: @izar Note that this change might require special merging with https://github.com/izar/pytm/pull/216

**Pull request merged**